docs-src/docs/reference/terraform-plugin.md

```diff
@@ -141,6 +143,10 @@ Available instance sizes can be found [here](https://aws.amazon.com/ec2/instance
 * `min_count` - (Optional, Default: 0) The minimum number of instances allowed in the environment
 * `user-data` - (Optional) The user data template to use for the environment's autoscaling group.
 See the [cli reference](/reference/cli/#environment) for the default template.
+* `os` - (Optional, Default: "linux") Specifies the type of operating system used in the environment.
+Options are "linux" or "windows".
+* `ami` - (Optional) Specifies the AMI to use in the environment. 
```

Comment:
Once more re: AMI ID vs name - I know down on line 160 it specifies "ID" but it probably wouldn't hurt to say so in a few places, and I don't think it's mentioned at all in the CLI reference.